### PR TITLE
wip: validate

### PR DESCRIPTION
**Pull request by macrozone**

### CODE REVIEW

File: packages/uniforms/src/ValidatedForm.tsx [hunk 1]
```diff
@@ -67,6 +67,7 @@ export function Validated<Base extends typeof BaseForm>(Base: Base) {
       return {
         ...super.getContext(),
         validating: this.state.validating,
+        validate: this.state.validate
```
Comment (macrozone) [inline packages/uniforms/src/ValidatedForm.tsx:70]: i am not sure if `validate` describes what it means

File: packages/uniforms/src/types.ts [hunk 2]
```diff
@@ -25,6 +25,7 @@ export type Context<Model> = {
   };
   submitting: boolean;
   validating: boolean;
+  validate?: boolean;
```
Comment (macrozone) [inline packages/uniforms/src/types.ts:28]: not sure if other forms should have this property too

**Comment by radekmie** [comment]
Hi @macrozone. I've added my comment in https://github.com/vazco/uniforms/issues/889#issuecomment-789663975 - please do let me know there.

Separately, please do fill the PR description and add a more descriptive title.

**Comment by macrozone** [comment]
superseed by https://github.com/vazco/uniforms/pull/906

**Pull request closed without merging**